src/Core/Commander/Providers/OGCWebServiceHelper.js

```diff
@@ -37,43 +38,50 @@ export default {
         texture.anisotropy = 16;
 
         return promise.then(() => {
-            cache.addRessource(url, texture);
+            if (!cache.getRessource(url)) {
+                cache.addRessource(url, texture);
+            }
             return texture;
         });
     },
-    getXBilTextureByUrl(url, pitch) {
+    getXBilTextureByUrl(url) {
         const textureCache = cache.getRessource(url);
 
         if (textureCache !== undefined) {
-            const { min, max } = ioDXBIL.computeMinMaxElevation(
-                textureCache.image.data,
-                SIZE_TEXTURE_TILE, SIZE_TEXTURE_TILE,
-                pitch);
-            return Promise.resolve({ pitch, textureCache, min, max });
+            return Promise.resolve({ texture: textureCache });
         }
 
-        return ioDXBIL.read(url).then((result) => {
+        const promiseXBil = (cachePending[url] || ioDXBIL.read(url)).then((result) => {
             // TODO  RGBA is needed for navigator with no support in texture float
             // In RGBA elevation texture LinearFilter give some errors with nodata value.
             // need to rewrite sample function in shader
+
+            // loading concurrence
+            const textureConcurrence = cache.getRessource(url);
+            if (textureConcurrence) {
+                return textureConcurrence;
+            }
+
             result.texture = getTextureFloat(result.floatArray);
             result.texture.generateMipmaps = false;
             result.texture.magFilter = THREE.LinearFilter;
             result.texture.minFilter = THREE.LinearFilter;
-            result.pitch = pitch;
-
             cache.addRessource(url, result.texture);
 
             return result;
         });
+
+        cachePending[url] = promiseXBil;
```

Comment:
`cachePending` will grow indefinitely.
You could probably remove elements from it after the line:
```js
cache.addRessource(url, result.texture);
```